word-guess-main.rb

```diff
@@ -0,0 +1,86 @@
+
+
+class Game
+
+  def initialize(input_word, word_descriptions)
+    @word = input_word.chars.to_a
+    @hidden_word = @word.join.gsub(/[A-Za-z]/, "_")
+    @bad_guesses = 0
+    @word_descriptions = word_descriptions
```

Comment:
I'm not sure I agree with calling this instance variable `@word_descriptions`. From the way it's used, it looks like it isn't a collection of descriptions, but a single description of the current word. `@word_description` (without the "s") might be a better choice.